core/endorser/endorser.go

```diff
@@ -111,8 +111,14 @@ func (e *Endorser) callChaincode(txParams *ccprovider.TransactionParams, input *
 		logger.Infof("finished chaincode: %s duration: %dms", chaincodeName, elapsedMillisec)
 	}(time.Now())
 
+	meterLabels := []string{
+		"channel", txParams.ChannelID,
+		"chaincode", chaincodeName,
```

Comment:
Are we sure we want to have this chaincode as well ? I thought that the general rule of thumb is not have something in the labels that has a large domain? 

pinging @sykesm , wdyt?

Reply:
Seems consistent with other metrics in this file.

Reply:
Indeed!